ui-core/src/styles/inputs/fieldWrapper.css

```diff
@@ -28,5 +32,53 @@
         margin-left: 0.563rem;
       }
     }
+
+    input:focus:has(.status-message-wrapper-tooltip) .status-message-wrapper-tooltip {
+      z-index: 2;
+    }
+
+    .status-message-wrapper-tooltip {
+      position: absolute;
+      display: flex;
+      align-items: center;
+      width: 320px;
+      top: 60px;
+      gap: 12px;
+      border-radius: 8px;
+      box-shadow:
+        0px 10px 20px 0px rgba(0, 0, 0, 0.2),
+        0px 1px 0px rgba(255, 255, 255, 0.6) inset;
+      background-color: theme('colors.error.5');
+      opacity: 1;
+      padding: 8px 12px;
```

Comment:
Not quite the values in mockup : 
![Capture d’écran 2024-12-20 à 09 24 22](https://github.com/user-attachments/assets/cc778817-60d5-4401-b337-d4779c3e0e2d)

```suggestion
      padding: 6px 12px 10px;
```